src/MysticMind.PostgresEmbed/PgServer.cs

```diff
@@ -44,6 +44,7 @@ public class PgServer : IDisposable
 
         private readonly Policy _downloadRetryPolicy;
         private readonly Policy _deleteFoldersRetryPolicy;
+        public string _nugetPackage { get; private set; }
```

Comment:
You can make this as `private readonly`

Reply:
fixed the "private readonly"; sorry, my fault.

I sincerely can't understand where the change is breaking, since the default value doesn't change the previous behaviour at all.

Regarding the other NuGet package to use, as soon as you (hopefully! :) accept the PR I will grab the simple yet powerful ps1 shell script by which the "standard" package is created and I will create my own nuget package with the latest postgresql versions that are currently missing.

Reply:
Ignore my comment on the breaking change, you have added a new constructor `public PgBinariesLiteBinaryDownloader(string pgVersion, string destDir, string nugetPackage)`, it should be fine.

Reply:
Thanks a lot!

Now, while you're drinking the coffee I just bought you, can you please update the nuget package too?! :)

Reply:
@mattiamerzi Really appreciate that you have bought me a coffee. I have been doing OSS work for close to 10 years and you are the first person to have bought me a coffee till date.

Quick note, I will release it is NuGet in few hours.

Reply:
@mattiamerzi I am have done refactoring etc, will need to test a bit hence will release during my day i.e +8hrs or so.